EIPS/eip-5252.md

```diff
@@ -0,0 +1,237 @@
+---
+eip: 5252
+title: Account-bound Finance
+description: An EIP-5114 extension that aids in preventing arbitrary loss of funds
+author: Hyungsuk Kang (@hskang9), Viktor Pernjek (@smuxx)
+discussions-to: https://ethereum-magicians.org/t/pr-5252-discussion-account-bound-finance/10027
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-06-29
+requires: 20, 721, 1155, 5114
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type of account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP extends [EIP-5114](./eip-5114.md) to further enable transferring fund to other accounts for mobility between managing multiple wallets(i.e. private keys).
+
+## Motivation
+
+Finance in the crypto ecosystem is facing a huge trust issue. Smart contracts are often proxies, with the actual logic of the contract hidden away in a separate logic contract. Many projects include a multi-signature "wallet" with unnecessarily-powerful permissions. And it is not possible to independently verify that stablecoins have enough real-world assets to continue maintaining their peg, creating a large loss of funds (such as happened in the official bankruptcy announcement of Celsius and UST de-pegging and anchor protocol failure). One should not trust exchanges or other third parties with one's own investments with the operators' background, status, or cult of personality in web3.0.
+
+The pattern empowers more self-sovereignty and gives more credentials(obviously with an Account-bound token) than locking financial data in the operating team's contract with a trustless way.
+
+#### Offchain-identity vs Soul-bound token on credentials 
+
+This EIP opposes the offchain-identity solution to take over the whole system, because their backend eventually relies on the trust of the operator, not the cryptographic proof of storing data(e.g. Proof-of-work, Proof-of-stake, etc). Offchain identity as credentials denies the whole premise of crypto's existence. Hence, Soul-bound token is the verifiable credential to stay, and offchain integration is just enough to show images.
```

Comment:
```suggestion
This EIP opposes the off-chain identity solution to take over the whole system because their backends eventually rely on the trust of the operator, not cryptographic proof (e.g. Proof-of-work, Proof-of-stake, etc). Off-chain identity as credentials denies the whole premise of crypto's existence. Soulbound tokens are a better, verifiable, credential, and data stored off-chain is only to store token metadata.
```
Required: make this clearer
Recommended: use this rephrasing

Reply:
It's just my personal opinion. Feel free to disregard it.

I will say, it actually does happen to be in Grammarly's dictionary - but I agree that that's hardly an authoritative source.